src/tests/files/bagh.json

```diff
@@ -159,7 +159,7 @@
         "display": "id",
         "additionalRelations": {
           "stadsdeel": {
-            "table": "stadsdeel",
+            "name": "stadsdelen",
```

Comment:
Mogen we de `key` van `additionalRelations` hier niet gebruiken? Zo werkt `additionalFilters` bvb

Reply:
Dat zou ook kunnen, maar dan is de lookup van de relatie minder efficient. Je moeten dan alle additionalRelations inspecteren en kijken of daar een "table" is die gelijk is aan de camelName

Reply:
Maar dat is misschien beter. Anders table is 2 keer gedefinerd.

Reply:
Die table kan weg want die data staat er dubbel. 

Maar ik zal nog een testcase toevoegen met bijvoorbeeld bezoekAdres en PostAdres relaties naar een Bedrijf en dan kijken of dat backwards goed gaat.